environments/staging/public.yml

```diff
@@ -130,6 +130,11 @@ localsettings:
   J2ME_ADDRESS: "{{ J2ME_SITE_HOST }}"
   LOCAL_APPS:
     - {name: 'ddtrace.contrib.django', host: webworkers}
+  LOCAL_MIDDLEWARE:
+    - 'django.middleware.security.SecurityMiddleware'
+  SECURE_HSTS_SECONDS: 300
+  SILENCED_SYSTEM_CHECKS:
+    - 'security.W008'
```

Comment:
From https://docs.djangoproject.com/en/3.1/ref/checks/:
> security.W008: Your SECURE_SSL_REDIRECT setting is not set to True. Unless your site should be available over both SSL and non-SSL connections, you may want to either set this setting to True or configure a load balancer or reverse-proxy server to redirect all connections to HTTPS.

Could you remind me why we want `SECURE_SSL_REDIRECT` to be `False`? Is it that we're just not ready to take that step yet (in which case maybe we let that warning stand), or that it doesn't apply to us / we don't intend to ever set it.

In case it's the problem described below, I'm pasting it, copied from https://docs.djangoproject.com/en/3.1/ref/settings/:

> If turning this to True causes infinite redirects, it probably means your site is running behind a proxy and can’t tell which requests are secure and which are not. Your proxy likely sets a header to indicate secure requests; you can correct the problem by finding out what that header is and configuring the SECURE_PROXY_SSL_HEADER setting accordingly.

But you know what, yeah maybe this doesn't matter at all since, the proxy we have in front of it already turns away HTTP requests and redirects them to HTTPS. Maybe just leave a comment to that effect.

Reply:
>But you know what, yeah maybe this doesn't matter at all since, the proxy we have in front of it already turns away HTTP requests and redirects them to HTTPS. Maybe just leave a comment to that effect.

Followed this line of thinking and added a comment to explain why we can safely silence this warning.